lms/templates/view.html

```diff
@@ -39,7 +39,7 @@ <h1>תצוגת תרגיל {{ solution['exercise']['id'] }}: {{ solution['exercis
   <div id="view-body">
     <div class="code-view-container">
       <div id="code-view" data-id="{{ solution['id'] }}" data-file="{{ current_file.id }}" data-exercise="{{ solution['exercise']['id'] }}" data-role="{{ role }}">
-        <pre><code id="python-user-code" class="language-python line-numbers highlight">{{- current_file.code | trim(chars=' ') | e -}}</code></pre>
```

Comment:
In my.js file there is we get this element by id of python-user-code.
Need to change there also the id.

Reply:
Awesome catch, 10x